recipes/cosmosis/meta.yaml

```diff
@@ -18,9 +18,7 @@ requirements:
   build:
     - python
     - pip
-    - {{ compiler('c') }}
-    - {{ compiler('fortran') }}
-    - {{ compiler('cxx') }}
+    - compilers
     - liblapack
     - libblas
     - {{ mpi }}
```

Comment:
Ack Sorry I was not clear. This section should only be

```
    - {{ compiler('c') }}
    - {{ compiler('fortran') }}
    - {{ compiler('cxx') }}
```

Reply:
The `compilers` package is for use in runtime environments and adds extra symlinks to standard names for the compilers which people expect.

Reply:
I went ahead and made this change.